Data/ByteString/Streaming/Internal.hs

```diff
@@ -378,24 +383,25 @@ foldlChunks f z = go z
         go a (Go m)       = m >>= go a
 {-# INLINABLE foldlChunks #-}
 
+-- | Instead of mapping over each `Word8` or `Char`, map over each strict
+-- `B.ByteString` chunk in the stream.
 chunkMap :: Monad m => (B.ByteString -> B.ByteString) -> ByteString m r -> ByteString m r
 chunkMap f bs = dematerialize bs return (Chunk . f) Go
 {-# INLINE chunkMap #-}
 
+-- | Like `chunkMap`, but map effectfully.
 chunkMapM :: Monad m => (B.ByteString -> m B.ByteString) -> ByteString m r -> ByteString m r
 chunkMapM f bs = dematerialize bs return (\bs' bss -> Go (fmap (`Chunk` bss) (f bs'))) Go
 {-# INLINE chunkMapM #-}
 
+-- | Like `chunkMapM`, but discard the result of each effectful mapping.
 chunkMapM_ :: Monad m => (B.ByteString -> m x) -> ByteString m r -> m r
 chunkMapM_ f bs = dematerialize bs return (\bs' mr -> f bs' >> mr) join
 {-# INLINE chunkMapM_ #-}
 
-
-{- | @chunkFold@ is preferable to @foldlChunks@ since it is
-     an appropriate argument for @Control.Foldl.purely@ which
-     permits many folds and sinks to be run simulaneously on one bytestream.
-
-  -}
+-- | @chunkFold@ is preferable to @foldlChunks@ since it is an appropriate
+-- argument for @Control.Foldl.purely@ which permits many folds and sinks to be
```

Comment:
```suggestion
-- | 'chunkFold' is preferable to 'foldlChunks' since it is an appropriate
-- argument for 'Control.Foldl.purely' which permits many folds and sinks to be
```